src/registrar/tests/test_reports.py

```diff
@@ -785,7 +785,7 @@ def test_domain_request_data_full(self):
                 "Chief Tester,Purpose of the site,CISA-first-name CISA-last-name | There is more,"
                 "Testy Tester testy2@town.com,"
                 "cisaRep@igorville.gov,city.com,\n"
-                "city6.gov,Submitted,Federal,Executive,Portfolio 1 Federal Agency,,N/A,,,2,,,,0,1,city1.gov,,,,,"
+                "city6.gov,Submitted,Federal,Executive,Portfolio 1 Federal Agency,,N/A,,NY,2,,,,0,1,city1.gov,,,,,"
```

Comment:
This changed because the test portfolio did not have a state_territory and the test domain did have a state_territory. When the report was modified to export city and state from domain, I needed to update the expected output in the test